nodeJS/express/deployment.md

```diff
@@ -0,0 +1,308 @@
+
+- You get a free one-time grant of $5 on their free trial, and the applications are never put to sleep when inactive.
+- However, the longevity of your free allowance depends on how many resources you consume. More complex apps with more traffic may consume all free resources within a month, whereas simpler apps may last longer.
+
+##### Railway.app: Links
+
+- [Railway homepage](https://railway.app/)
+- [Railway documentation](https://docs.railway.app/)
+
+---
+
+#### Adaptable.io
+
+- Like Railway, has a convenient deployment process. You link to your project's GitHub repo.
+- Free plan does not limit the number of applications you can deploy.
+- Also has fixed and usage-based payment plans.
+
+##### Adaptable.io: Free Plan
+
+- No limits on the number of applications you can deploy on the free plan.
+- Monthly performance allowance is more than sufficient for course/personal projects (approximately 25,000 API requests per month).
+- Applications are put to sleep when inactive but wake up speed is quicker than Render.
+- Requires a credit card.
+
+##### Adaptable.io: Links
+
+- [Adaptable.io homepage](https://adaptable.io/)
+- [Adaptable.io documentation](https://adaptable.io/docs/what-is-adaptable)
+- [Guide: Official getting started with deploying an Express app on Adaptable guide](https://adaptable.io/docs/app-guides/deploy-express-app)
+
+---
+
+#### Render
+
+- Render allows you to deploy using "Blueprints", which link to your project's GitHub repo.
+- The free 750-hour allowance is enough to host a few apps without paying anything. However, databases are separate on Render, and the lowest spec databases cost $7 each.
+- $21 a month is enough to host three applications as each app's database will cost $7.
+
+##### Render: Free Plan
+
+- 750 hours of free usage per month.
+- Applications are put to sleep automatically after 15 minutes of inactivity, so the 750 free hours should be enough to host a few apps for the entire month.
+- Requires a credit card.
+
+##### Render: Links
+
+- [Render homepage](https://render.com/)
+- [Render documentation](https://render.com/docs/)
+- [Guide: Official getting started with Node/Express on Render guide](https://render.com/docs/deploy-node-express-app)
+- [Guide: CodeBrah video guide for deploying NodeJS applications to Render](https://www.youtube.com/watch?v=bnCOyGaSe84&ab_channel=CodeBrah)
+
+---
+
+#### Koyeb
+
+- Easily Git push to deploy.
+- Ability to run web apps, APIs, and workers easily.
+- SSL and 10 free custom domains
+
+##### Koyeb: Free Plan
+
+- One free Web service.
+- One free Postgres database (50 hours).
+- Doesn't require a credit card to start.
+- Pay-per-use by the second for more.
+
+##### Koyeb: Links
+
+- [Koyeb homepage](https://www.koyeb.com/)
+- [Koyeb documentation](https://www.koyeb.com/docs)
+- [Guide: Official getting started with Node/Express on Koyeb guide](https://www.koyeb.com/docs/deploy/express)
+
+---
+
+#### Neon
+
+- Database branching
+-
```

Comment:
Rogue line?
```suggestion
```

Reply:
Yea basically a placeholder to indicate more information should probably go here